### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by hramezani**
# Pull requests

Thank you for submitting a contribution to Django Translations.

Please ensure the following steps have been taken:

## All changes

- [ ] Is this contribution for a known issue in the issues? If so mention it.
- [ ] Is there a task for this contribution in the project board? If so mention it.

## Code changes

- [ ] Have you run the unit tests on the project before submitting the contribution?
- [ ] Have you added the appropriate unit tests for your contribution to the project?
- [ ] Have you added the appropriate documentation for your contribution to the project?
- [ ] Have you styled your code properly based on the contribution guidelines?

## Documentation changes

- [ ] Have you built the documentation?
- [ ] Have you run the documentation tests?

## Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by bbmokhtari** [comment]
Thank you for putting in the time to create a pull request for Django-Translations.
All in all it's OK, but I think it needs some very minor changes.

File: .github/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ There are two types of content, each of which has its own challenges for transla
   _e.g. "Please enter a valid email address."_
 
   Django already provides a
-  [solution](https://docs.djangoproject.com/en/2.1/topics/i18n/translation/)
+  [solution](https://docs.djangoproject.com/en/3.1/topics/i18n/translation/)
```
Comment (bbmokhtari) [inline .github/README.md:17]: I this this should remain the lowest possible version to ensure backwards-compatibility.
So `2.2` for now.

File: .travis.yml [hunk 2]
```diff
@@ -211,16 +139,6 @@ jobs:
         - pip install sphinx
       before_script: python create.py && python config.py
       script: make --directory docs doctest
-    - name: "Python 3.6, Django 2.1, SQLite - Doc Test"
```
Comment (bbmokhtari) [inline .travis.yml:214]: The test for "Python 3.6, Django 2.0, SQLite - Doc Test" should also be removed. (`stage` should remain as you know)

File: .travis.yml [hunk 3]
```diff
@@ -311,16 +209,6 @@ jobs:
         - pip install sphinx
       before_script: python create.py && python config.py
       script: make --directory docs doctest
-    - name: "Python 3.8, Django 2.1, SQLite - Doc Test"
```
Comment (bbmokhtari) [inline .travis.yml:314]: The test for "Python 3.8, Django 2.0, SQLite - Doc Test" should also be removed.

File: .travis.yml [hunk 4]
```diff
@@ -407,7 +275,7 @@ jobs:
       env: EXAMPLE_ENGINE=sqlite3
       before_install: skip
       install:
-        - pip install django~=2.0
+        - pip install django~=3.0
```
Comment (bbmokhtari) [inline .travis.yml:278]: Please allow this to remain the lowest possible version for ensuring the backwards-compatibility issue mentioned above, meaning `2.2`.

File: requirements_dev.txt [hunk 5]
```diff
@@ -1,4 +1,4 @@
-Django~=2.0
+Django~=3.0
```
Comment (bbmokhtari) [inline requirements_dev.txt:1]: Please allow this to remain the lowest possible version for ensuring the backwards-compatibility issue mentioned above, meaning `2.2`.

**Review by bbmokhtari — CHANGES_REQUESTED** [review]
Please check the PR code comments.
Also feel free to discuss any of them with me if you have another opinion.

**Comment by hramezani** [comment]
Thanks @bbmokhtari for the review. I've addressed all your comments.

**Review by bbmokhtari — APPROVED** [review]

**Comment by bbmokhtari** [comment]
Approved and merged. Thank you.

**Pull request merged**